src/main/java/com/example/pizzapp/repositories/OrderRepository.java

```diff
@@ -0,0 +1,17 @@
+package com.example.pizzapp.repositories;
+
+import com.example.pizzapp.models.Order;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+import java.util.Optional;
+
+@Repository
+public interface OrderRepository extends JpaRepository<Order, Long> {
+
+    List<Orders> findAllByUserId(Long id);
```

Comment:
```suggestion
    List<Orders> findAllByUserId(Long userId);
```

тут сам неправильно написал на прошлом ревью, так что извиняюсь )